### PR TITLE
Reimplement Only-count-Natural-Spawned-mobs-towards-natural-spawn patch

### PR DESCRIPTION
**Pull request by kickash32**
This patch was removed due to changes in 1.14 MC, I have reimplemented it so it works in 1.14 MC.

### CODE REVIEW

File: Spigot-Server-Patches/0402-Only-count-Natural-Spawned-mobs-towards-natural-spaw.patch [hunk 1]
```diff
@@ -0,0 +1,60 @@
+From 13cdbbfbad353171b8f8b20f1d9785f24246de96 Mon Sep 17 00:00:00 2001
+From: kickash32 <kickash32@gmail.com>
```
Comment (aikar) [inline Spigot-Server-Patches/0402-Only-count-Natural-Spawned-mobs-towards-natural-spaw.patch]: Please restore original patch headers with me as author and original date.

**Review by aikar — CHANGES_REQUESTED** [review]

File: Spigot-Server-Patches/0402-Only-count-Natural-Spawned-mobs-towards-natural-spaw.patch (hunk 1 above)
Comment (kickash32) [reply Spigot-Server-Patches/0402-Only-count-Natural-Spawned-mobs-towards-natural-spaw.patch]: Is this good?

Comment (kashike) [reply Spigot-Server-Patches/0402-Only-count-Natural-Spawned-mobs-towards-natural-spaw.patch]: Missing original date too.

**Comment by kashike** [comment]
@kickash32 You seem to only be modifying the patch in the `removed` directory now.

**Comment by kickash32** [comment]
> @kickash32 You seem to only be modifying the patch in the `removed` directory now.

no its actually moving the patch, the pathnames are too long to be visible on GitHub though

**Review by kashike — APPROVED** [review]

**Pull request closed without merging**